app/assets/javascripts/talking_stick/talking_stick.js.erb

```diff
@@ -124,11 +124,12 @@ var TalkingStick = (function(self) {
     }
   };
 
-  self.connect = function() {
+  self.connect = function(name) {
```

Comment:
Note to self: this probably should be moved out of TalkingStick proper, since it's more a concern of the signaling later.  I'm concerned too that the `name` data has nothing to do with the needs of connecting. At the very least, it should be a hash of data.  I'll go ahead and merge this since you put it in the correct place, but it was a mistake I started earlier. I'll come back and refactor this.